packages/artifactory-ha/artifactory-ha.patch

```diff
@@ -1,10 +1,32 @@
 diff -x '*.tgz' -x '*.lock' -uNr packages/artifactory-ha/charts-original/Chart.yaml packages/artifactory-ha/charts/Chart.yaml
 --- packages/artifactory-ha/charts-original/Chart.yaml
 +++ packages/artifactory-ha/charts/Chart.yaml
-@@ -24,3 +24,6 @@
+@@ -16,3 +16,6 @@
  - https://bintray.com/jfrog/product/JFrog-Artifactory-Pro/view
  - https://github.com/jfrog/charts
- version: 3.0.14
+ version: 4.7.6
 +annotations:
 +  catalog.cattle.io/certified: partner
 +  catalog.cattle.io/release-name: artifactory-ha
+diff -x '*.tgz' -x '*.lock' -uNr packages/artifactory-ha/charts-original/Chart.yaml.orig packages/artifactory-ha/charts/Chart.yaml.orig
+--- packages/artifactory-ha/charts-original/Chart.yaml.orig
++++ packages/artifactory-ha/charts/Chart.yaml.orig
+@@ -0,0 +1,18 @@
++apiVersion: v1
++appVersion: 7.12.6
++description: Universal Repository Manager supporting all major packaging formats,
++  build tools and CI servers.
++home: https://www.jfrog.com/artifactory/
++icon: https://raw.githubusercontent.com/jfrog/charts/master/stable/artifactory-ha/logo/artifactory-logo.png
++keywords:
++- artifactory
++- jfrog
++- devops
++maintainers:
++- email: installers@jfrog.com
++  name: Chart Maintainers at JFrog
++name: artifactory-ha
++sources:
++- https://bintray.com/jfrog/product/JFrog-Artifactory-Pro/view
++- https://github.com/jfrog/charts
++version: 4.7.6
```

Comment:
This shouldn't be in the patch. Is likely that when you changed the url in `package.yaml` to your new upstream, the patch didn't apply cleanly and these were created. Please remove it if you patch is working properly. This applies to https://github.com/rancher/partner-charts/pull/73 as well

Reply:
@PennyScissors Thanks for your review. This issue has been fixed in this PR and also in #73 . Please check.